### PR TITLE
Improve error details for strange multi-active encounter situation.

### PR DESCRIPTION
**Pull request by pbugni**
Also, don't generate a stack, but log the error and march on.

### CODE REVIEW

File: portal/models/user.py [hunk 1]
```diff
@@ -362,8 +362,15 @@ def current_encounter(self):
...
-        return query.one()
+        if query.count() != 1:
+            # Not good - we should only have one `active` encounter for
+            # the current user.  Log details for debugging and return the
+            # first
+            msg = "Multiple active encounters found for {}: {}".format(
+                self.user,
+                [(e.status, str(e.start_time), str(e.end_time)) for e in query])
+            current_app.logger.error(msg)
+        return query.first()
```
Comment (ivan-c) [inline portal/models/user.py]: Would it make sense to sort by creation time of encounter? Not sure if `first()` would get oldest or newest encounter...

Comment (pbugni) [inline portal/models/user.py]: No idea - this is an unexpected error situation.  Which would be the correct one to return :)

Comment (ivan-c) [inline portal/models/user.py]: Good point. I wanted the Encounter returned to be consistent (either first or last) but that doesn't really matter...

**Comment by ivan-c** [comment]
Looks good, not sure what else we could log...

**Comment by ivan-c** [comment]
Merged into develop too FYI

**Pull request merged**